### DISCUSSION TITLE
/opt/install/installDBBinaries.sh: line 55: /opt/install/database/runInstaller: No such file or directory

### DISCUSSION THREAD

**Discussion by gowseshaik** [oracle/docker-images#2067] [Q&A]
error:
/opt/install/installDBBinaries.sh: line 55: /opt/install/database/runInstaller: No such file or directory
--> 853a6c19758
[3/3] STEP 1/9: FROM 795a5d573f57c90bfe48b0aa286f7cef99e32ab611bcbe8e16d588e890af5611
[3/3] STEP 2/9: USER oracle
--> 0f3b22d87c5
[3/3] STEP 3/9: COPY --chown=oracle:dba --from=builder $ORACLE_BASE $ORACLE_BASE
--> 75cd5e3f070
[3/3] STEP 4/9: USER root
--> 546063b35ab
[3/3] STEP 5/9: RUN $ORACLE_BASE/oraInventory/orainstRoot.sh &&     $ORACLE_HOME/root.sh
/bin/sh: /opt/oracle/oraInventory/orainstRoot.sh: No such file or directory
Error: error building at STEP "RUN $ORACLE_BASE/oraInventory/orainstRoot.sh &&     $ORACLE_HOME/root.sh": error while running runtime: exit status 127
 gouse  /  data  OracleDBContainer                                                                                                                    

My env's in Dockerfile
ENV ORACLE_BASE=/opt/oracle \
    ORACLE_HOME=/opt/oracle/product/12.2.0.1/dbhome_1 \
    INSTALL_DIR=/opt/install \
    INSTALL_FILE_1="linuxx64_12201_database.zip" \
    INSTALL_RSP="db_inst.rsp" \
    CONFIG_RSP="dbca.rsp.tmpl" \
    PWD_FILE="setPassword.sh" \
    RUN_FILE="runOracle.sh" \
    START_FILE="startDB.sh" \
    CREATE_DB_FILE="createDB.sh" \
    SETUP_LINUX_FILE="setupLinuxEnv.sh" \
    CHECK_SPACE_FILE="checkSpace.sh" \
    CHECK_DB_FILE="checkDBStatus.sh" \
    USER_SCRIPTS_FILE="runUserScripts.sh" \
    INSTALL_DB_BINARIES_FILE="installDBBinaries.sh"

What could be the issue,

**Comment by yunus-qureshi** [comment]
@gowseshaik seems like a DB installation issue
did you download linuxx64_12201_database.zip from http://www.oracle.com/technetwork/database/enterprise-edition/downloads/index.html  and place it under dockerfiles/12.2.0.1
